packages/invoices/src/OutboundShipment/DetailView/reducer.ts

```diff
@@ -252,19 +261,62 @@ export const reducer = (
             line
           );
 
-          if (existingSummaryItem && existingRow) {
-            existingRow.isUpdated = existingRow.isCreated ? false : true;
-            existingRow.isDeleted = false;
-            existingRow.numberOfPacks = line.numberOfPacks;
+          // If the row is being updated
+          if (existingSummaryItem) {
```

Comment:
what a mess... 😢

Reply:
good to get it in and then we can have a go at refactoring! though it's not bad and is clear enough to read